src/fsharp/TastReflect.fs

```diff
@@ -988,9 +988,10 @@ and ReflectAssembly(g, ccu: CcuThunk, location:string) as asm =
                 | None -> failwith (sprintf "TODO: didn't get back to CCU being compiled for local tcref %s" tcref.DisplayName)
             let reflAssem = ccuofTyconRef.ReflectAssembly :?> ReflectAssembly
             let tcrefR = reflAssem.TxTypeDef None tcref
-            match tinst with 
-            | [] -> tcrefR 
-            | args -> tcrefR.MakeGenericType(Array.map asm.TxTType (Array.ofList args))  
+            tcrefR
+            //match tinst with 
```

Comment:
Yeah, see the later commit. I haven't had any time :( since then thou so comments still stand.